src/main/java/io/github/thebusybiscuit/slimefun4/api/events/RadiationDamageEvent.java

```diff
@@ -0,0 +1,71 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+/**
+ * The {@link RadiationDamageEvent} is called when a player takes radiation damage.
+ * 
+ * @author HoosierTransfer
+ *
+ */
+public class RadiationDamageEvent extends Event implements Cancellable {
+    private static final HandlerList handlers = new HandlerList();
+
+    private final Player player;
+    private int exposure;
```

Comment:
This isn't final so I assume it's meant to be mutable, but I don't see a setExposure method